moto/core/models.py

```diff
@@ -12,11 +12,27 @@
 class MockAWS(object):
     nested_count = 0
 
-    def __init__(self, backends):
+    def __init__(self, backends, fixtures={}):
```

Comment:
It is generally best to not use [mutable default arguments](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)